src/state.py

```diff
@@ -1,3 +1,6 @@
+"""
+Global runtime variables.
+"""
```

Comment:
Here you can also transform comments into docstrings. See http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autodata